packages/ag-charts-community/src/util/json.test.ts

```diff
@@ -197,40 +198,29 @@ describe('json module', () => {
                     c: [10, 9, 8, 7, 6],
                 };
 
-                const merge = jsonMerge([base, mergee1, mergee2]);
+                const merge = mergeDefaults(mergee2, mergee1, base);
                 expect(merge).toMatchSnapshot();
                 expect(merge).toHaveProperty('a', mergee2.a);
                 expect(merge).toHaveProperty('b', mergee1.b);
                 expect(merge).toHaveProperty('c', mergee2.c);
             });
 
             it('should merge arrays correctly', () => {
-                const base: any = [[{ x: 1 }, { y: 1 }], [{ m: 2, n: 2 }]];
-                const mergee1: any = [];
-                const mergee2: any = [[{ x2: 1 }, { y2: 1 }], [{ m2: 2, n2: 2 }]];
+                const base: any = { a: [[{ x: 1 }, { y: 1 }], [{ m: 2, n: 2 }]] };
+                const mergee1: any = { a: [] };
+                const mergee2: any = { a: [[{ x2: 1 }, { y2: 1 }], [{ m2: 2, n2: 2 }]] };
 
-                const merge = jsonMerge([base, mergee1, mergee2]);
+                const merge = mergeDefaults(mergee2, mergee1, base);
                 expect(merge).toMatchSnapshot();
                 expect(merge).toEqual(mergee2);
             });
 
-            it('should honour DELETE value', () => {
```

Comment:
This case was to support users unsetting options on update, are you happy that this scenario is catered for?

Reply:
It was used only to remove properties from results and wasn't exported to the users but only internally.
I personally think it was an additional mechanism that just added unnecessary complexity, instead I just "manually" ignore specific properties in the 2-4 places where it was required.